packages/synapse-react-client/src/components/DataGrid/DataGridTypes.ts

```diff
@@ -1,3 +1,24 @@
+import { Model } from 'json-joy/lib/json-crdt'
+import { s } from 'json-joy/lib/json-crdt-patch'
+
+const gridRowSchema = s.obj({
+  data: s.vec(s.con('')),
+  metadata: s.obj({
+    synapseRow: s.obj({
+      rowId: s.con(0),
+      versionNumber: s.con(0),
+      etag: s.con(''),
+    }),
+  }),
+})
+
+const gridSchema = s.obj({
```

Comment:
```suggestion
const _gridSchema = s.obj({
```

Nit: rename to _gridSchema to avoid [no-unused-vars rule](https://typescript-eslint.io/rules/no-unused-vars/): `'gridSchema' is assigned a value but only used as a type`.